### PR TITLE
Develop timeline

### PR DESCRIPTION
**Pull request by ben2collins**

### CODE REVIEW

File: idfive-component-library/src/components/ks/graph/graph.scss [hunk 1]
```diff
@@ -0,0 +1,154 @@
+@include media('table') {
```
Comment (drogers98) [inline idfive-component-library/src/components/ks/graph/graph.scss]: isnt this part of your other PR ??

File: idfive-component-library/src/scss/index.scss [hunk 2]
```diff
@@ -37,6 +37,8 @@
 @import "../components/ks/table/table";
 @import "../components/ks/video/video";
 @import "../components/ks/parallax/parallax";
+@import "../components/ks/timeline/timeline";
+@import "../components/ks/graph/graph";
```
Comment (drogers98) [inline idfive-component-library/src/scss/index.scss]: isnt this part of your other PR?

**Review by drogers98 — CHANGES_REQUESTED** [review]
check graph stuff, you have it added here as well as the other PR.

File: idfive-component-library/src/scss/index.scss (hunk 2 above)
Comment (ben2collins) [reply idfive-component-library/src/scss/index.scss]: Yes this part of the Timeline Module PR

File: idfive-component-library/src/components/ks/graph/graph.scss [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+                .svg-inline--fa {
+                    width: 4em;
+                    height: 4em;
+                }
+            }
+        }
+    }
+
+
+        .hp_graph_bundle {
```
Comment (drogers98) [inline idfive-component-library/src/components/ks/graph/graph.scss]: you will need to do the indent here as well if you are including on this PR vas well as the other. Or, remove graph stuff from this pr

**Pull request closed without merging**